app/Modules/Api/V1/Model/EventApiService.php

```diff
@@ -0,0 +1,53 @@
+<?php declare(strict_types=1);
+
+namespace App\Modules\Api\V1\Model;
+
+use App\Modules\Core\Model\Entity\Event;
+use App\Modules\Core\Model\EventModel;
+use Nette\Application\LinkGenerator;
+use Nette\Utils\DateTime;
+
+class EventApiService
+{
+	/** @var EventModel */
+	private $eventModel;
+
+	/** @var LinkGenerator */
+	private $linkGenerator;
+
+	public function __construct(EventModel $eventModel, LinkGenerator $linkGenerator)
+	{
+		$this->eventModel = $eventModel;
+		$this->linkGenerator = $linkGenerator;
+	}
+
+	public function getEvents()
+	{
+		$events = [];
+
+		foreach ($this->eventModel->getAllWithDates([], new DateTime()) as $eventRow) {
+			$event = Event::createFromRow($eventRow);
+			$eventTags = [];
+			foreach ($this->eventModel->getEventTags($eventRow) as $eventTag) {
+				$eventTags[] = [
+					'id' => $eventTag->getTag()->getId(),
+					'rate' => $eventTag->getRate(),
+					'name' => $eventTag->getTag()->getName(),
+					'code' => $eventTag->getTag()->getCode(),
+				];
+			};
+			$events[] = [
+				'id' => $event->getId(),
```

Comment:
Should omit `id`. We would use some hash for events details reference anyway.
Don't forget update Apiary.